sharpy/linear/src/lin_aeroelastic.py

```diff
@@ -46,7 +46,7 @@ class LinAeroEla():
         SS (scipy.signal): state space formulation (discrete or continuous time), as selected by the user
     """
 
-    def __init__(self, data, custom_settings_linear=None, uvlm_block=False):
+    def __init__(self, data, custom_settings_linear=None, uvlm_block=False, chosen_ts=None):
```

Comment:
Thanks for this!

I'm going to add a setting to the `LinearAssembler` (if it is not already there) to use this as well. I will make a new commit before we merge